src/main/java/nl/dtls/fairdatapoint/Profiles.java

```diff
@@ -0,0 +1,35 @@
+/**
+ * The MIT License
+ * Copyright © 2017 DTL
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package nl.dtls.fairdatapoint;
+
+public class Profiles {
+
+    public static final String PRODUCTION = "production";
```

Comment:
This could use some documentation on how and where these can be used.

Reply:
Not sure, how to document it more. Profiles are standard mechanisms of how to distinguish different environments (https://docs.spring.io/spring-boot/docs/current/reference/html/boot-features-profiles.html). Or you mean to describe more where the production or development or testing is used?

Reply:
Partially yes, but for example newcomers to spring-boot might not be too familiar with the convention. A one-liner for each profile would help in that case, and potentially an `@see` link on the class level to the reference docs you linked.

Reply:
That would be nice for newcomers in Spring Boot or in Spring of course, but I am not sure if it is a good idea to have it there. Because then, we should add the same `@see` annotation to `@Service`, `@Configuration`, etc. 

The only thing (which is not covered in Spring Docs) is concrete profiles. I can add a one-line comment to each profile. But not sure if it helps (because the name seems very straightforward):
```
/**
* Profile which is used in production
*/
public static final String PRODUCTION = "production";

/**
* Profile which is used during development
*/
public static final String PRODUCTION = "production";
```